.github/workflows/e2e.yml

```diff
@@ -44,15 +44,24 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        cable-driver: ['libreswan', 'wireguard', 'vxlan']
-        extra-toggles: ['', 'globalnet', 'ovn']
+        cable-driver: ['', 'wireguard', 'vxlan']
+        extra-toggles: ['', 'ovn']
+        globalnet: ['', 'globalnet']
+        # Run most tests against the latest K8s version
+        k8s_version: ['1.28']
         exclude:
           - cable-driver: wireguard
             extra-toggles: ovn
         include:
           - extra-toggles: dual-stack
           - extra-toggles: dual-stack, globalnet
```

Comment:
Doesn’t this need to be changed to match the extraction of Globalnet from `extra-toggles` above?